src/components/CytoscapeGraph/CytoscapeGraph.tsx

```diff
@@ -333,15 +334,25 @@ export class CytoscapeGraph extends React.Component<CytoscapeGraphProps, Cytosca
       return;
     }
     const duration = this.props.graphDuration.value;
+    const healthPerNamespace = new Map<String, Promise<NamespaceHealth>>();
     // Asynchronously fetch health
     cy.nodes().forEach(ele => {
       const fqService = ele.data('service');
       if (fqService && (ele.data('isGroup') || !ele.data('parent'))) {
         const serviceParts = fqService.split('.');
+        if (serviceParts.length < 2) {
```

Comment:
It is enough to filter by length ?
is there any other special node besides "unknown" one ?

Reply:
I think "unknown" is the only one at this time, but since I'm extracting `serviceParts[0]` and `serviceParts[1]` below, this is a way to filter out "unknown" and run a defensive check at the same time.